packages/backend-core/src/environment.ts

```diff
@@ -82,6 +82,7 @@ const environment = {
   SESSION_UPDATE_PERIOD: process.env.SESSION_UPDATE_PERIOD,
   DEPLOYMENT_ENVIRONMENT:
     process.env.DEPLOYMENT_ENVIRONMENT || "docker-compose",
+  LOG_4XX: process.env.LOG_4XX,
```

Comment:
This naming makes sense, but could be confusing to self hosters and it could be a very useful feature, a slightly more verbose name could be beneficial? It doesn't describe if it enables/disables etc.

Reply:
Added here: https://github.com/Budibase/budibase/commit/91e63c8ec41fdc865b98eb114eedd26ea3e289a4